src/services/leafletMarkersHelpers.js

```diff
@@ -101,8 +106,8 @@ angular.module('ui-leaflet').service('leafletMarkersHelpers', function ($rootSco
     };
 
     var _resetMarkerGroup = function (groupName) {
-        if (isDefined(groups[groupName])) {
-            groups.splice(groupName, 1);
+        if (angular.isDefined(groups[groupName])) {
```

Comment:
It is in scope with the function. Just stick to the patterns that the library already uses please. Which is making vars in scope from leafletHelpers. There is very little going on in terms of global scope. Like you said global scope is normally either L or angular thats it.